recipe/meta.yaml

```diff
@@ -17,19 +17,19 @@ requirements:
   build:
     - cmake
     - python
-    - numpy 1.8.*
+    - numpy
     - jasper
-    - libpng >=1.6.32,<1.6.35
-    - libnetcdf 4.5.*
-    - hdf5 1.10.1
+    - libpng
+    - libnetcdf
+    - hdf5
     - ecmwf_grib {{ version }}
   run:
     - python
-    - numpy >=1.8
+    - numpy
     - jasper
-    - libpng >=1.6.32,<1.6.35
-    - libnetcdf >=4.5
-    - hdf5 1.10.1
+    - libpng
```

Comment:
libpng, jasper, libnetcdf and hdf5 all have run_exports and are not needed in the run section.  Python and the pin_compatible(numpy) need to stay in run.

Reply:
Got it. (I'll use this to write some docs on recipe migration. Thanks!)

BTW conda-forge versions for `libpng`, `jasper`, `libnetcdf` and `hdf5` do not have `run_exports` yet, not sure if that is a safe move until we do.

Reply:
I would recommend doing those other recipes first. Adding in lower level run exports later requires a lot more recipe editing.

Without the upstream run exports, you need equivalent pin_compatible lines for each of these.

Reply:
Sure. This one is just an experiment and I'm not planning to merge it anytime soon.